### PR TITLE
Versi chainspec based on Rococo.

### PR DESCRIPTION
**Pull request by eskimor**
Fixes https://github.com/paritytech/polkadot/issues/4481

### CODE REVIEW

**Review by bkchr — APPROVED** [review]

**Comment by PierreBesson** [comment]
@eskimor can you regenerate the Versi json chainspec. As it currently is just a copy-paste of the rococo one.

File: cli/src/command.rs [hunk 1]
```diff
@@ -167,6 +171,7 @@ impl SubstrateCli for Cli {
 	}
 
 	fn native_runtime_version(spec: &Box<dyn service::ChainSpec>) -> &'static RuntimeVersion {
+		dbg!(spec.id());
```
Comment (drahnr) [inline cli/src/command.rs:174]: Stray leftover?

**Review by drahnr — APPROVED** [review]
One stray `dbg!(_)`, other than that LGTM

File: cli/src/command.rs [hunk 2]
```diff
@@ -159,6 +171,7 @@ impl SubstrateCli for Cli {
...
```
Comment (bkchr) [inline cli/src/command.rs:174]:
```suggestion
```

**Comment by drahnr** [comment]
bot merge

**Pull request merged**